src/components/ResourceTypePicker/index.js

```diff
@@ -40,27 +40,15 @@ const ResourceTypePicker = ({
   resourceTypeFilters,
   selectResourceTypeAction,
   selectedResourceType,
-}) => {
-  // select first resourceType on initial load
-  useEffect(() => {
-    let firstEnabledType;
-    resourceTypeFilters.forEach(({ type, typeIsEnabled }) => {
-      if (!firstEnabledType && typeIsEnabled) {
-        firstEnabledType = type;
-      }
-    });
-    selectResourceTypeAction(firstEnabledType);
-  }, []);
-
-  return (
-    <View>
-      <ScrollView
-        style={styles.root}
-        horizontal
-        showsHorizontalScrollIndicator={false}
-        contentContainerStyle={styles.contentContainerStyle}
-      >
-        {
+}) => (
+  <View>
+    <ScrollView
+      style={styles.root}
+      horizontal
+      showsHorizontalScrollIndicator={false}
+      contentContainerStyle={styles.contentContainerStyle}
+    >
+      {
           resourceTypeFilters
```

Comment:
should the reducer `createCollection` be updated to have a non-null `selectedResourceType` ?